src/App.js

```diff
@@ -1,23 +1,127 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+
+import babyNamesData from "./babyNamesData.json";
+import BabyNamesCard from "./BabyNamesCard";
+import SearchBar from "./SearchBar";
+import FavoriteNames from "./FavoriteNames";
+import ResetButton from "./ResetButton";
+import { useEffect, useState } from "react";
+import GenderSelector from "./GenderSelector";
 
 function App() {
+  let [favoriteName, setFavoriteName] = useState([]);
+  let [noneFavoriteNames, setNoneFavoriteNames] = useState(babyNamesData);
+  let [isClickingFavoriteNames] = useState(true);
+  const [showFavoriteNameMessage, setShowFavoriteNameMessage] = useState(true);
```

Comment:
You're using a mixture of `let` and `const` for your state variables -- in this case they can all be `const`.

Reply:
great suggestion thank you